test/parallel/test-async-wrap-trigger-id.js

```diff
@@ -13,16 +13,12 @@ process.nextTick(() => {
     triggerAsyncId1 = triggerAsyncId();
     assert.notStrictEqual(
       triggerAsyncId0,
-      triggerAsyncId1,
-      'Async resources having different causal ancestry ' +
-      'should have different triggerAsyncIds');
```

Comment:
This text being deleted is value. If we don't want it in a message, let's at least preserve it as a code comment.